src/fusor/models.py

```diff
@@ -27,7 +30,8 @@ class FUSORTypes(str, Enum):
     TRANSCRIPT_SEGMENT_ELEMENT = "TranscriptSegmentElement"
     TEMPLATED_SEQUENCE_ELEMENT = "TemplatedSequenceElement"
     LINKER_SEQUENCE_ELEMENT = "LinkerSequenceElement"
-    GENE_ELEMENT = "GeneElement"
+    # TODO: I'm not sure if this needs to still be here or not
+    GENE = "Gene"
```

Comment:
yeah I think you can revert this change, we still need the GeneElement that wraps `domain_models.Gene`